Sources/AppcuesKit/Presentation/Traits/Appcues/ExperienceWrapperViewController.swift

```diff
@@ -18,12 +18,11 @@ internal class ExperienceWrapperViewController<BodyView: ExperienceWrapperView>:
     // Only set for bottom-aligned modals. Need access to the constant value for keyboard avoidance.
```

Comment:
Should be fine, but worth double checking that slideouts work nicely with the keyboard showing?

Reply:
good call, looks like its all good

https://github.com/appcues/appcues-ios-sdk/assets/19266448/46f18d01-442c-431f-a2d7-602f0420ea02